packages/back-end/src/api/members/updateMemberRole.ts

```diff
@@ -0,0 +1,185 @@
+import { isRoleValid, roleSupportsEnvLimit } from "shared/permissions";
+import { cloneDeep } from "lodash";
+import { orgHasPremiumFeature } from "enterprise";
+import { updateOrganization } from "../../models/OrganizationModel";
```

Comment:
We should start using absolute paths:
```suggestion
import { updateOrganization } from "@back-end/src/models/OrganizationModel";
```

Reply:
Let's hold off on using absolute paths for now since there are some issues with our prod builds.